sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -18,11 +18,11 @@
     "- Connect to your AML workspace from the Python SDK\n",
     "- Create an `AutoML time-series forecasting Job` with the 'forecasting()' factory-fuction.\n",
     "- Train the model using AmlCompute by submitting/running the AutoML forecasting training job\n",
-    "- Obtaing the model and score predictions with it\n",
+    "- Obtain the model and score predictions with it\n",
```

Comment:
Please change to: Obtain the model and use it to generate forecast.

Reply:
Fixed.